Makefile

```diff
@@ -1,5 +1,5 @@
 # Current Operator version
-OPERATOR_VERSION ?= 0.15.3
+OPERATOR_VERSION ?= 0.16.0
```

Comment:
could you please update the opa https://github.com/observatorium/opa-openshift/commit/4860be8ed8df4485c124d0927c0a7a0c4e68999a ?